v2/apiserver/internal/core/kubernetes/substrate.go

```diff
@@ -568,3 +655,622 @@ func (s *substrate) countRunningPods(
 	}
 	return len(podsSet), nil
 }
+
+func (s *substrate) createWorkspacePVC(
+	ctx context.Context,
+	project core.Project,
+	event core.Event,
+) error {
+	storageQuantityStr := event.Worker.Spec.WorkspaceSize
+	if storageQuantityStr == "" {
+		storageQuantityStr = "1G"
```

Comment:
Should this be a constant somewhere?

Reply:
Thought about it, but I don't know what benefit we'd get from it.